File: iCovieApi/iCovieApi/Models/Master/SignupModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class SignUpModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string firstname { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string lastname { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string emailid { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string profileimg { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int issubmitsigneddoc { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int isapprovedmembersipdoc { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int isnotifyapprovalemailsms { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}

File: iCovieAdmin-Ng/New folder/src/app/core/auth/_models/Masters/signup.model.ts

export class SignupModel {
    id: number;
    companyid: number;
    locationid: number;
    membershipid: string;
    first_name: string;
    lastn_ame: string;
    email_id: string;
    is_submit_signed_doc: number;
    is_approve_memebership: number;
    is_notify_approval_email_sms: number;
   
    cid: number;
    isCoreCategory: boolean = false;
    clear(): void {
        this.id = undefined;
        
        this.companyid=undefined;
        this.locationid=undefined;
        this.membershipid='';
        this.first_name='';
        this.lastn_ame='';
        this.email_id='';
        this.is_submit_signed_doc=undefined;
        this.is_approve_memebership=undefined;
        this.is_notify_approval_email_sms=undefined;
        
        this.cid = undefined;
        this.isCoreCategory = false;
    }
}

